js/scripts.js

```diff
@@ -0,0 +1,14 @@
+var allSpansEven = $("span:even");
+    allSpansEven.each(function(index, element) {
+    	$(allSpansEven).css('color', 'red');
```

Comment:
znowu w środku pętli mówisz 40 razy "weż wszystkie parzyste spany i je pokoloruj" - koloruj pojedynczo albo nie stosuj pętli